src/main/java/org/folio/edge/sip2/MainVerticle.java

```diff
@@ -64,12 +66,13 @@ public void start(Future<Void> startFuture) {
           new FolioResourceProviderModule(config().getString("okapiUrl"), vertx),
           new ApplicationModule());
       handlers = new EnumMap<>(Command.class);
-      handlers.put(LOGIN, injector.getInstance(LoginHandler.class));
-      handlers.put(CHECKIN, injector.getInstance(CheckinHandler.class));
       handlers.put(CHECKOUT, injector.getInstance(CheckoutHandler.class));
```

Comment:
I was ordering it to follow the SIP spec order. I think I had it that way in a few other places and wanted the order to be consistent. For some reason I thought it would make it easier to see if we were missing something. Now that I think about it, it is probably a dumb reason...

Reply:
No worries. Just want to make sure that I'm not missing out on any important implicit concepts.